app/views/rooms/show.html.erb

```diff
@@ -120,17 +120,49 @@
         </div>
         <hr>
 
-         <div class="row">
-            <div class="col-md-12">
-                <h3> 0 Reviews <span id="average_rating"></span></h3>
-                <div class="container">
-                     <div class="text-center">
-                         <h4>There are no reviews.</h4>
+          <div class="row">
+            <div class="col-md-12">                
+                <% if @reviews.count == 0 %>
```

Comment:
you can check the reviews count by `@reviews.any?`

Reply:
This wont work on my side.